packages/kit/src/views/Send/pages/SendConfirm/TxExtraInfoContainer.tsx

```
@@ -21,14 +21,24 @@ import {
   const intl = useIntl();
   const [shouldShowData, setShouldShowData] = useState<boolean>(false);
   const { copyText } = useClipboard();
 
-  const encodedTx = unsignedTx?.encodedTx as IEncodedTxEvm;
+  const dataContent = useMemo(() => {
+    if (!unsignedTxs || unsignedTxs.length === 0) {
+      return '';
+    }
+    return unsignedTxs.reduce((acc, unsignedTx) => {
+      const tx = unsignedTx.encodedTx as IEncodedTxEvm;
+      if (tx && tx.data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)